modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FrontendDataSetAdmin.macro

```diff
@@ -159,6 +167,36 @@ definition {
 		}
 	}
 
+	macro createMultipleFilters {
+		for (var key_filterType : list ${key_filterList}) {
+			LexiconEntry.gotoAdd();
+
+			Select(
+				locator1 = "FrontendDataSetAdmin#FILTER_BY_SELECT",
+				value1 = ${key_filterType});
+
+			Click(
+				key_text = "Save",
+				locator1 = "CP2ActivationKeys#KEY_DEACTIVATION_TERMS");
+		}
+	}
+
+	macro createSingleFilter {
+		LexiconEntry.gotoAdd();
+
+		Type(
+			locator1 = "TextInput#NAME",
+			value1 = ${filterName});
+
+		Select(
+			locator1 = "FrontendDataSetAdmin#FILTER_BY_SELECT",
+			value1 = ${key_filterType});
+
+		Click(
+			key_text = "Save",
+			locator1 = "CP2ActivationKeys#KEY_DEACTIVATION_TERMS");
+	}
+
```

Comment:
This macro is not needed, you can just use createMultipleFilters() and pass just one parameter to it. Also, as you will use just this macro, I would rename it to createFilters()